src/hooks/use-graph-line-styles.ts

```diff
@@ -1,33 +1,31 @@
+import { useMemo } from "react";
 import { GraphLineScheme } from "../settings";
 
 export interface GraphLineStyles {
-  x: undefined | string;
-  y: undefined | string;
-  z: undefined | string;
+  x: undefined | [number, number];
```

Comment:
I think the [number, number] type deserves a comment explaining what they represent or a named type.